### ISSUE TITLE
Misleading behavior while deleting a livechat message

### ISSUE THREAD

**Issue by janrudolph** [RocketChat/Rocket.Chat#7698]
### Description:

If an agent deletes a message of an ongoing livechat conversation, the message is not deleted in the customer's livechat inteface. However, this might be a feature: Since the message was already sent to the customer, a deletion might not make sense. The customer already read the message.
Regarding the current implementation, an agent might think the mesasge was deleted and is not accessible by the customer anymore. But this is not true.

In my opinion, there are some possible solutions for this issue, e.g.

1. Do not allow to delete livechat messages.
2. Allow deletion, but inform the agent, that the message has not been deleted in the customer's livechat interface. 
3. Allow deletion and delete the message in the customer's livechat interface 

The final solution should be configurable in the admin interface. 

### Server Setup Information:

* Version of Rocket.Chat Server: 0.56.0
* Operating System: Linux
* Deployment Method(snap/docker/tar/etc): AWS
* Number of Running Instances: 2
* Node Version: v4.8.2

### Steps to Reproduce:

1. Start new livechat by customer
2. Chat some time (customer & agent)
3. Agent deletes one message by selecting the option at a message
4. Message is deleted in the backend but not in the front end.

### Expected behavior:

see description

### Actual behavior:

see description

**Comment by ashwaniYDV** [comment]
I could not produce this issue.
There is no option to delete livechat messages by the agent.

**Comment by sampaiodiego** [comment]
@renatobecker is this still a thing?

**Comment by renatobecker** [comment]
> @renatobecker is this still a thing?

@sampaiodiego yes. I just checked the code of the Livechat widget and the fact is that our `JS/SDK` lust provide the `onMessage` event, which has been used to propagate new messages from the server to the widget, but the `SDK` doesn't provide any event handler to listen when a message is removed.

So, yeah this case still happens, we need to implement a solution for it.
I can handle it, I'll put it on our short-time roadmap.

Thanks.
